config/reference_config.go

```diff
@@ -141,7 +141,12 @@ func (refconfig *ReferenceConfig) Refer() {
 	}
 
 	//create proxy
-	refconfig.pxy = extension.GetProxyFactory(consumerConfig.ProxyFactory).GetProxy(refconfig.invoker, url)
+	if refconfig.Async {
```

Comment:
do you can add a test for `Test_ReferAsync` ?

Reply:
ok,i will add test for this method ASAP